src/property-filter/__tests__/property-filter-token-editor.test.tsx

```diff
@@ -342,256 +344,338 @@ const defaultEditorProps: TokenEditorProps = {
+          'i18nStrings.tokenEditorTokenRemoveLabel': 'Remove filter',
+          'i18nStrings.tokenEditorTokenRemoveFromGroupLabel': 'Remove filter from group',
+          'i18nStrings.tokenEditorAddNewTokenLabel': 'Add new filter',
+          'i18nStrings.tokenEditorAddTokenActionsAriaLabel': 'Add filter actions',
+          'i18nStrings.tokenEditorAddExistingTokenAriaLabel': 'Add filter {token__formattedText} to group',
+          'i18nStrings.tokenEditorAddExistingTokenLabel':
+            'Add filter {token__propertyLabel} {token__operator} {token__value} to group',
+        },
+      }}
+    >
+      <TokenEditorWithEmptyI18n {...props} />
+    </TestI18nProvider>
+  );
+}
+
+function renderTokenEditor(props?: Partial<TokenEditorProps>, withProvider = false) {
+  const Component = withProvider ? TokenEditorWithProvider : TokenEditorWithI18n;
+  const { container } = render(<Component {...defaultEditorProps} {...props} />);
+  return new InternalPropertyFilterEditorDropdownWrapper(container);
+}
 
-  test.each([false, true])('renders token editor with a single property, supportsGroups=%s', supportsGroups => {
-    const wrapper = renderTokenEditor({ supportsGroups });
-    const propertyField = wrapper.findPropertyField();
-    const operatorField = wrapper.findOperatorField();
-    const valueField = wrapper.findValueField();
-    const removeActions = wrapper.findTokenRemoveActions();
-    const addActions = wrapper.findTokenAddActions();
-
-    // Not supported yet because the standalone editor does not include the popover
-    // This can be uncommented when defining the test against the property filter
-    // which is not possible for the internal-only token-editor-grouped yet.
-    // expect(wrapper.findHeader().getElement()).toHaveTextContent('Edit token');
-    // expect(wrapper.findDismissButton().getElement()).toHaveAccessibleName('Close token editor');
-
-    // Token inputs
-    expect(propertyField.findControl()!.find('button')!.getElement()).toHaveAccessibleName('Property Name');
-    expect(operatorField.findControl()!.find('button')!.getElement()).toHaveAccessibleName('Operator =');
-    expect(valueField.findControl()!.find('input')!.getElement()).toHaveAccessibleName('Value');
-    expect(valueField.findControl()!.find('input')!.getElement()).toHaveValue('John');
-
-    if (supportsGroups) {
-      expect(removeActions!.findNativeButton().getElement()).toHaveAccessibleName('Remove actions Name = John');
-      expect(removeActions!.findNativeButton().getElement()).toBeDisabled();
-
-      expect(addActions!.findNativeButton().getElement()).toHaveAccessibleName('Add filter actions');
-      expect(addActions!.findNativeButton().getElement()).toBeDisabled();
-      expect(addActions!.findMainAction()!.getElement()).toHaveTextContent('Add new filter');
-    } else {
-      expect(removeActions).toBe(null);
-      expect(addActions).toBe(null);
+describe.each([false, true])('with i18n-provider %s', withProvider => {
```

Comment:
That is the actual change done to this file. Now all tests are run with i18nStrings and with I18nProvider to ensure the labels can be passed in either way.